lib/sequenceserver/makeblastdb.rb

```diff
@@ -161,7 +161,7 @@ def determine_unformatted_fastas
         next if @formatted_fastas.any? { |f| f[0] == path }
 
         @fastas_to_format << [path,
-          make_db_title(File.basename(path)),
+          make_db_title(path.clone),
```

Comment:
Why do you need to clone path?

Reply:
@yeban because on super `make_db_title` method, i saw there is methods 'tr!', and 'gsub!', which is replace the current `path` variable value, and it will affect next call method `guess_sequence_type_in_fasta` and return an error. Should I clone it to different variable?

Reply:
Hmm, what you need is “db_name = Fule.basename(path)” as first line of “make_db_title” method. Also, “db_name” argument of that method should be renamed to “path”.

Reply:
I see, I've updated the commit
Thanks